search-app/src/cmr/search/api/tags.clj

```diff
@@ -20,10 +20,15 @@
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
-(defn- tag-api-response
+(defn- has-error?
+  "Whether there's an error in [data]."
+	[data]
+  (some (fn [element] (contains? element "errors")) data))
+
+(defn tag-api-response
   "Creates a successful tag response with the given data response"
   ([data]
-   (tag-api-response 200 data))
+    (if (has-error? data) (tag-api-response 400 data) (tag-api-response 200 data)))
```

Comment:
Fix this formatting like so:
`if <condition>`
`statement to evaluate if true`
`statement to evaluate if false`
```suggestion
    (if (has-error? data) 
      (tag-api-response 400 data)
      (tag-api-response 200 data)))
```